File: Iserv.Niis.Angular/src/app/shared/models/edit-payment-use-response-dto.ts
export class EditPaymentUseResponseDto {
    amountIsGreaterThanPaymentInvoiceReminder: boolean;
    paymentInvoiceReminder: number;
    amountIsGreaterThanPaymentReminder: boolean;
    paymentReminder: number;
    paymentInvoiceNewReminderIsGreaterThan100KZT: boolean;
    success: boolean;
}

File: Iserv.Niis.Model/Models/Payment/EditPaymentUseResponseDto.cs
namespace Iserv.Niis.Model.Models.Payment
{
    public class EditPaymentUseResponseDto
    {
        public bool AmountIsGreaterThanPaymentInvoiceReminder { get; set; }
        public decimal PaymentInvoiceReminder { get; set; }
        public bool AmountIsGreaterThanPaymentReminder { get; set; }
        public decimal PaymentReminder { get; set; }
        public bool PaymentInvoiceNewReminderIsGreaterThan100KZT { get; set; }
        public bool Success { get; set; }
    }
}